File: apzkr-pzpi-21-10-karasov-mykhailo/Task4-MobileClient/app/src/main/java/com/example/tasksyncmobileapp/model/dto/RegistrationDto.kt
package com.example.tasksyncmobileapp.model.dto

import android.text.Editable
import java.util.Date

data class RegistrationDto(
    val email: String,
    val password: String,
    val passwordConfirm: String,
    val firstName: String,
    val secondName: String,
    val birthday: String,
    val phoneNumber: String
)

File: apzkr-pzpi-21-10-karasov-mykhailo/Task1-Server/src/core/repositories/AuthRepository/dto/RegistrationDto.ts
export class RegistrationDto {
    constructor(
        public readonly email: string,
        public readonly password: string,
        public readonly passwordConfirm: string,
        public readonly firstName: string,
        public readonly secondName: string,
        public readonly birthday: Date,
        public readonly phoneNumber: string | null,
    ) {}
}